addOns/websocket/src/main/javahelp/org/zaproxy/zap/extension/websocket/resources/help/contents/script.html

```diff
@@ -20,5 +20,11 @@ <h2>Fuzzer WebSocket Processor</h2>
 Fuzzer WebSocket Processor scripts are called before forwarding a fuzzed WebSocket message to the client/server. The payloads will have already been injected at this point.<br>
 A template script is provided which gives details of the methods and parameters supported. 
 
+<h2>WebSocket Passive Scan</h2>
+
+WebSocket Passive Scan scripts are called every time a WebSocket message frame is transmitted over a WebSocket connection. Script can access WebSocket messages in order to examine the payload and raise an alert.<br>
```

Comment:
“...WebSocket connection. Script can...” &gt; “...WebSocket connection. Scripts can...” (Plural on scripts)

Reply:
Fixed. Thank you!